bloom-housing/bloom
# feat: add address holder fields to paper application

# Pull Request Template

## Issue Overview

This PR addresses #3705 #3693


## Description

Adds address holder name and relationship fields to paper application. Also even thought address was required to submit if present, it wasn't highlighting errors, so i fixed that. 

This PR was affecting also Application summary, so to avoid wrong labels i connected it with #3693.

## How Can This Be Tested/Reviewed?

On partners go to listing -> applications tab. If db freshly seeded editing/adding application should work as before. Changing settings -> preferences -> options to collect address and then selecting yes on address holder name and/or relationship will show the fields in paper application when preference option checked. Also select "verify address by radius", and pick radius value, so you can test all 3 validation variants. 

1. In paper application you should now be able to add name and relationship to preference option (and then it should be visible in summary)
2. To see address verification field in summary you need to fill digital application.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
